osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-dsmr22.json

```diff
@@ -576,6 +700,306 @@
+      "group": "ABSTRACT",
+      "meterTypes": ["SP","PP"],
+      "properties": {},
+      "attributes": [
+        {
+          "id": 2,
+          "description": "buffer",
+          "datatype": "array",
+          "valuetype": "DYNAMIC",
+          "value": "EMPTY",
+          "access": "R"
+        },
+        {
+          "id": 3,
+          "description": "capture objects",
+          "datatype": "array",
+          "valuetype": "FIXED_IN_PROFILE",
+          "value": "CLOCK,2|ACTIVE_ENERGY_IMPORT_RATE_1,2|ACTIVE_ENERGY_IMPORT_RATE_2,2|ACTIVE_ENERGY_EXPORT_RATE_1,2|ACTIVE_ENERGY_EXPORT_RATE_1,2|MBUS_MASTER_VALUE,2",
+          "access": "R"
+        },
+        {
+          "id": 4,
+          "description": "capture period in sec",
+          "datatype": "double-long-unsigned",
+          "valuetype": "FIXED_IN_PROFILE",
+          "value": "0",
+          "access": "R"
+        },
+        {
+          "id": 5,
+          "description": "sort method",
+          "datatype": "enum",
+          "valuetype": "FIXED_IN_METER",
+          "value": "1",
+          "access": "R"
+        },
+        {
+          "id": 6,
+          "description": "sort object",
+          "datatype": "object_definition",
+          "valuetype": "FIXED_IN_METER",
+          "value": "NONE",
+          "access": "R"
+        },
+        {
+          "id": 7,
+          "description": "entries in use",
+          "datatype": "double-long-unsigned",
+          "valuetype": "DYNAMIC",
+          "value": "0",
+          "access": "R"
+        },
+        {
+          "id": 8,
+          "description": "profile entries",
+          "datatype": "double-long-unsigned",
+          "valuetype": "FIXED_IN_PROFILE",
+          "value": "13",
+          "access": "R"
+        }
+      ]
+    },
+    {
+      "tag": "AMR_PROFILE_STATUS_HOURLY_G",
+      "description": "AMR Profile Status Code M-Bus",
+      "class-id": 1,
+      "version": 0,
+      "obis": "0.x.96.10.3.255",
+      "group": "GAS",
+      "meterTypes": ["SP","PP"],
+      "properties": {},
+      "attributes": [
+        {
+          "id": 2,
+          "description": "value",
+          "datatype": "unsigned",
+          "valuetype": "DYNAMIC",
+          "value": "0",
+          "access": "R"
+        }
+      ]
+    },
+    {
+      "tag": "AMR_PROFILE_STATUS",
+      "description": "AMR Profile Status Code E meter",
+      "class-id": 1,
+      "version": 0,
+      "obis": "0.0.96.10.2.255",
+      "group": "ABSTRACT",
+      "meterTypes": ["SP","PP"],
+      "properties": {},
+      "attributes": [
+        {
+          "id": 2,
+          "description": "value",
+          "datatype": "unsigned",
+          "valuetype": "DYNAMIC",
+          "value": "0",
+          "access": "R"
+        }
+      ]
+    },
+    {
+      "tag": "MBUS_MASTER_VALUE",
+      "description": "M-Bus volume hourly",
+      "note": null,
+      "class-id": 4,
+      "version": 0,
+      "obis": "0.x.24.2.1.255",
```

Comment:
Ik zie deze code niet in de spec staan

Reply:
It's in the DSMR2.2 P3 spec as `0-x:24.2.e.255 `. Where `e` means instance number. The spec doesn't explain what value to use for `e`, but since DSMR4.2.2 uses `0.x.24.2.1.255` that seems to be the best choice.